src/Integration.Vsix/Commands/HelpMenu/AboutCommand.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SonarLint.VisualStudio.Core;
+
+namespace SonarLint.VisualStudio.Integration.Vsix.Commands
+{
+    internal class AboutCommand : VsCommandBase
+    {
+        private readonly string aboutUrl = "https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarlint-vscode";
```

Comment:
LEYLA!!!
For shame.

![image](https://user-images.githubusercontent.com/60586879/205322512-e1b92b26-8fca-4e4d-aefe-38b7d2bd8aa2.png)